.github/dependabot.yml

```diff
@@ -10,6 +10,11 @@ updates:
   schedule:
     interval: weekly
   open-pull-requests-limit: 6
+  groups:
```

Comment:
@travier I'm thinking we should add this as well to our repo templates for Rust-based projects.

Reply:
I based this on what I found in https://github.com/search?q=path%3A.github%2Fdependabot.yml+-is%3Afork+cargo+group+minor&type=code bTW

Reply:
> @travier I'm thinking we should add this as well to our repo templates for Rust-based projects.

Yeah, let's see how it works here.
bootc is using renovate right now, although I think our instance is broken, which I need to look at